lib/translate/BUILD

```diff
@@ -0,0 +1,12 @@
+cc_library(
+  name = "inquiry_lib",
```

Comment:
In some other translation prs, we mentioned having all the translate files be under one `translate_lib` libary